node_active: false
debug: false
publish_frequency: 4.0
fixed_frame_id: "map"
base_frame_id: "base_footprint"
pos_source: "amcl" #amcl or odom
v_threshold: 0.2
w_threshold: 0.2
time_interval: 0.1
path_length: 30
linear_acclrt: 0.1
angular_acclrt: 0.1

